build.gradle.kts

```diff
@@ -38,34 +38,31 @@ dependencies {
     testImplementation(libs.truth)
 }
 
+val targetJavaVersion = JavaVersion.VERSION_1_8
+val targetKotlinVersion = "1.6"
```

Comment:
Upgrading the Kotlin plugin to the latest but continuing to build with compatibility to the oldest supported Kotlin version. I'm not sure if this is required for Gradle plugins but it feels like a safer change.